scripts/spack/packages/mfem/package.py

```diff
@@ -7,19 +7,18 @@
 import shutil
 import sys
 
-from spack import *
+from spack.package import *
```

Comment:
any word on MFEM adding a cmake-based variant?

Reply:
none that i have heard.